### DISCUSSION TITLE
'Auto save' option

### DISCUSSION THREAD

**Discussion by 4l3m4oO** [friction2d/friction#48] [Ideas]
If there's some option to controls when/where the software generates an auto-save file?!

**Comment by rodlie** [comment]
Was thinking about auto save yesterday, will need to figure out if it's going to be based on a timer, event or undo (?)

**Reply by AlmostMikhail** [reply]
Timer would be good

**Reply by rodlie** [reply]
Current timer is 300000 msec. There is no UI to change this, but you can manually set it in the config.

```
[files]
AutoSaveTimeout=INT
```

**Reply by pgilfernandez** [reply]
And what about a Preferences option to set the time? There may be people that even would like to disable it (setting it to 0)
